automation/BuildRegistrationDefinition/__version__.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+#
+# MIT License
+#
+# Copyright (c) 2021 International Business Machines
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+"""
+Includes title, description, URL, version, build, author/email, license,
+and copyright.
+"""
+
+__version__ = "1.0.0"
+__title__ = "BuildRegistrationDefinition"
+__description__ = (
+    "Build a Hyper Protect Virtual Server registration definition file"
+)
+__url__ = "https://github.ibm.com/fhe-experience/HiperProtectExperiments/automation/BuildRegistrationDefinition"
+__build__ = 0x00100
+__author__ = "Dan FitzGerald"
+__author_email__ = "danfitz@us.ibm.com"
```

Comment:
we might need to add a "__maintainer_ and put the active maintainer email here. I don't know as Dan will keep it going since others modified the script and are already maintaining it.